Description: Avoid explicit need of argparse, part of python3.2 on which we depend anyway.
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2012-05-18
--- a/setup.py
+++ b/setup.py
@@ -6,5 +6,4 @@
       author_email = 'max@xolus.net',
       url = 'http://bitbucket.org/maugier/shatag',
       packages = ['shatag','shatag.backend','shatag.store'],
-      install_requires=['argparse'],
       scripts = ['bin/shatag', 'bin/shatag-add', 'bin/shatag-mkdb', 'bin/shatagd'])
